mapbox_gl_platform_interface/lib/src/location.dart

```diff
@@ -38,10 +36,7 @@ class LatLng {
     return <double>[latitude, longitude];
   }
 
-  static LatLng _fromJson(dynamic json) {
-    if (json == null) {
-      return null;
-    }
+  static LatLng _fromJson(List<double> json) {
```

Comment:
Looks like this needs to be `List<dynamic>`

```
[VERBOSE-2:ui_dart_state.cc(186)] Unhandled Exception: type 'List<dynamic>' is not a subtype of type 'List<double>'
#0      CameraPosition.fromMap
package:mapbox_gl_platform_interface/src/camera.dart:65
#1      MethodChannelMapboxGl.updateMapOptions
package:mapbox_gl_platform_interface/src/method_channel_mapbox_gl.dart:159
<asynchronous suspension>
#2      MapboxMapController._updateMapOptions
package:mapbox_gl/src/controller.dart:274
<asynchronous suspension>
```

Reply:
Thank you, I corrected it.